v2.0/known-limitations.md

```diff
@@ -10,6 +10,10 @@ This page describes newly identified limitations in the CockroachDB v2.0 release
 
 ## New Limitations
 
+### Replication factor for system ranges
+
+Changes to the <code>.default</code> replication zone are not automatically applied to other <a href="configure-replication-zones.html#edit-the-replication-zone-for-a-system-range">pre-configured replication zones</a> for internal system data or to any <a href="configure-replication-zones.html#user-created-replication-zones">user-created replication zones</a>. If you increase the replication factor for <code>.default</code>, you may also want to increase the replication factor for <code>.meta</code>, <code>.liveness</code>, and <code>system.jobs</code> to ensure that important internal data is as resilient as your user data.
```

Comment:
Maybe we don't care to make the distinction, but this behavior (changes to `.default` not affecting pre-existing zones) has always been the case -- the new thing in v2.0 is that there are more pre-configured zones to be affected by the existing problem.

Reply:
When put like that, I don't think we need to call this out as a "limitation" per se. It's more of a ux-gotcha, so hopefully covering it in the replication zone docs is enough.